src/main/java/jenkins/plugins/slack/SlackNotifier.java

```diff
@@ -594,7 +656,13 @@ public SlackJobProperty(String teamDomain,
                                 boolean includeTestSummary,
                                 boolean showCommitList,
                                 boolean includeCustomMessage,
-                                String customMessage) {
+                                String customMessage,
```

Comment:
Oh wow, I can't believe this made it through. adding stuff to a deprecated class feels wrong 😅 @timja @KreAch3R

Reply:
SlackNotifier isn't deprecated, an old constructor is deprecated that's being maintained for compatibility